gui/wxpython/main_window/frame.py

```diff
@@ -637,16 +636,21 @@ def BuildPanes(self):
             .Name("python")
             .Caption("Python")
             .Right()
-            .Layer(2)
-            .Position(3)
             .BestSize(self.PANE_BEST_SIZE)
             .MinSize(self.PANE_MIN_SIZE)
             .CloseButton(False)
             .MinimizeButton(True)
             .MaximizeButton(True),
+            target=self._auimgr.GetPane("modules"),
         )
 
         self._auimgr.GetPane("toolbarNviz").Hide()
+
+        # Set Modules as active tab
+        modules = self._auimgr.GetPane("modules")
+        notebook = self._auimgr._notebooks[modules.notebook_id]
```

Comment:
You can get the notebooks like this:
```suggestion
        notebook = self._auimgr.GetNotebooks()[0]
```
but it's also not ideal.

Reply:
Not ideal, but better. :-)